cmd/youtubedr/downloader.go

```diff
@@ -90,3 +100,19 @@ func getVideoWithFormat(id string) (*youtube.Video, *youtube.Format, error) {
 
 	return video, format, nil
 }
+
+// filterCodecs filters out codec with strings.Contains and uses AND operator
+func filterCodecs(video *youtube.Video, codec []string) {
+	var formats youtube.FormatList
+VideoFormat:
+	for _, f := range video.Formats {
```

Comment:
We already have mime type filtering in `FormatList#FindByType`. Please try to use it.

Reply:
Thanks, I did not notice that method. I think of a method returning the formatlist so that we can use FormatList#FindByItag & FormatList#FindByQuality after filtering by MimeTypes. What do you think?